### PR TITLE
fix and improve messages in/out for addresses

### PR DESCRIPTION
**Pull request by grs**
Also change the description and add tooltip giving more info.

### CODE REVIEW

File: agent/www/components/addresses/addresses.html [hunk 1]
```diff
@@ -38,22 +38,44 @@
...
+            <strong>{{item.messages_in_delta}}</strong> Incoming Traffic
+          </span>
+        </div>
+        <div class="list-view-pf-additional-info-item">
+          <span title="Messages sent from this address in last 5 minutes">
+          <span class="fa fa-sign-out"></span>
+          <strong>{{item.messages_out_delta}}</strong> Outgoing Traffic
+          </span>
+        </div>
+        <!--
```
Comment (k-wall) [inline agent/www/components/addresses/addresses.html]: Commented out code.

File: agent/www/components/addresses/addresses.html [hunk 2]
```diff
@@ -38,22 +38,44 @@
         </div>
       </div>
       <div class="list-view-pf-additional-info">
+        <div class="list-view-pf-additional-info-item">
+          <span title="Messages sent to this address in last 5 minutes">
+            <span class="fa fa-sign-in"></span>
+            <strong>{{item.messages_in_delta}}</strong> Incoming Traffic
```
Comment (k-wall) [inline agent/www/components/addresses/addresses.html]: I'm not sure about "Incoming Traffic". I think the user will still be confused.   The hover over helps, but the user would need to know to look for this. I think we need to find a way to display a more descriptive field label. Maybe we could use smaller text beneath the value "Message In - Last 5 min" etc.

I think also the change needs to be reflected on the Connections tab too.

File: agent/www/components/addresses/addresses.html (hunk 1 above)
Comment (grs) [reply agent/www/components/addresses/addresses.html]: Fixed

File: agent/www/components/addresses/addresses.html (hunk 2 above)
Comment (k-wall) [reply agent/www/components/addresses/addresses.html]: > I think the connections tab is different. There the count is the actual number of messages on that connection.

I hadn't appreciated that those were absolute values.  Thanks.

Comment (k-wall) [reply agent/www/components/addresses/addresses.html]: @gsim - any thoughts on my comments on the label used on the address tab?

Comment (grs) [reply agent/www/components/addresses/addresses.html]: @k-wall I don't have strong feelings about the text. I think the time period it covers is really a detail. Though useful to be able to find it out easily, in my view it does not need to be part of the main label; perhaps adding an info icon as the target for the mouseover would help inform users of the extra text available.

Apart for the time period, 'message in' was what was there before which I thought was reported as being unclear.

**Comment by kornys** [comment]
@grs @k-wall I rebased this branch on current "fixed" master

**Comment by k-wall** [comment]
#1445 intended to complain that screen labels imply an all-time total for messages in and messages out for the address.  I think the label does need to communicate the real nature of the value somehow.  The user shouldn't need to hover/click/readthedocs to find this out.

I was wondering about adding the text "over the last 5 minutes" below the two values, spanning both columns to reduce the screen clutter.

For me the label using the word Messages is better than Traffic.

**Comment by grs** [comment]
@k-wall do you want to commit the text you feel is appropriate onto this PR then?

**Comment by k-wall** [comment]
@grs - here's what my suggestion looks like.
stackmessageinoutwithlegend

**Comment by grs** [comment]
@kwall to me the alignment looks a bit odd there, but I don't object if that is what is wanted, just an observation.

**Comment by k-wall** [comment]
I've got a really poor eye for aesthetics of a UI.   Are you referring to the alignment of the tick box on the left hand side?

**Comment by grs** [comment]
Yes, but also the stacked items themselves - to me - look odd in the flow. Again though, I'm not opposing the change, just commenting. It may well be that the extra clarity/directness outweighs any subjective opinions on aesthetics.

**Comment by k-wall** [comment]
As discussed, I value the extra clarity/directness.  If we can think of a way of improving the presentation, then we can improve later.

**Comment by RH-sdavey** [comment]
@k-wall  I have to agree with @grs a little with the aesthetics, the tick box on the left looks really out of sorts right now... 
brokeredui
compared to previous "cleaner" version
filter1
and the stacked stats boxes dont work so well imo ( it grates on me that the arrows both point the same way for messages in and messages out... ) , any chance to put them all inline with just the text "values over the last 5 minutes" underneath ( as it is now) 
or what about a little "popup" window with text displayed
 "all values displayed here are aggregated over the last 5 minutes" or something similar.
something like this example
hawtio
it could be shown just once on the first time the address tab is loaded (I suppose thats possible)

**Pull request merged**